src/hooks/video/agora/useAgoraCamera.ts

```diff
@@ -68,13 +66,13 @@ export const useAgoraCamera: ReactHook<
   };
 
   const leaveChannel = useCallback(async () => {
-    if (!localCameraTrack || !localMicrophoneTrack || !client) return;
+    if (!client) return;
 
-    localCameraTrack.stop();
-    localCameraTrack.close();
+    localCameraTrack?.stop();
+    localCameraTrack?.close();
 
-    localMicrophoneTrack.stop();
-    localMicrophoneTrack.close();
+    localMicrophoneTrack?.stop();
+    localMicrophoneTrack?.close();
 
     setLocalCameraTrack(undefined);
     setLocalMicrophoneTrack(undefined);
```

Comment:
Is this change correct? By guarding based on `!client` it means we're never going to update `localCameraTrack`, `localMicrophoneTrack`, `setLocalCameraTrack`, `setLocalMicrophoneTrack`; so I expect they are now going to be more likely to 'fall out of sync' and cause weird bugs.

Reply:
@0xdevalias fixed in 1a95556